example/feature_demo/demo_types_test.go

```diff
@@ -1,111 +0,0 @@
-package example
```

Comment:
Did this test get moved somewhere else and github lost it?

Reply:
probably got deleted. I'll do another lookaround to see what's gone